content/collections/data/en/automated-tasks-in-data-assistant.md

```diff
@@ -0,0 +1,100 @@
+---
+id: bef1ff7c-2c93-4677-b05e-1588205fbd4d
+blueprint: data
+title: 'Automated Tasks'
+this_article_will_help_you:
+  - 'Understand how automated tasks can help keep your data clean'
+  - 'Learn how to set up and run automated tasks'
+landing: false
+exclude_from_sitemap: false
+updated_by: 3f7c2286-b7da-4443-a04f-7c225af40254
+updated_at: 1754950505
+---
+Automated Tasks are part of Amplitude’s [Data Assistant](/docs/data/use-ai-data-assistant) feature. Automated tasks let data governors define conditions and actions that the system then performs automatically. It automates repetitive clean-up workflows by taking recommendations provided by the Data Assistant and transforming those recommendations into automatic actions. For example, an automated task can automatically delete events if they have been unused for 90 days. 
+
+After they have been created, automated tasks run daily. 
+
+Automated tasks is only available on Enterprise plans. Go to the [pricing](https://amplitude.com/pricing) page for more details.
+
+{{partial:admonition type="note" heading=""}}
+Automated tasks are only available for users with manager or admin permissions. Contact your administrator if you can't access Automated tasks as an option for Data Assistant.
+{{/partial:admonition}}
+
+You can access your automated tasks from the Data Assistant by going to *Data > Assistant > Automated Tasks*. 
+If you have active automations, those are displays along with a current count of the events affected by that specific automation. If you don't have any active automations, you'll have the opportunity to set them up. 
+
+By default, automated tasks operate across all projects within a workspace. You can, however, specify for the automation to only search through a single project.
+
+## Types of automated tasks
+
+You can select one, or any, of the following types of tasks to automate: 
+
+* Clean up stale events
+* Clean up single-day events
+* Clean up unused events
+
+### Clean up stale events
+This tasks removes events that haven't experiences any recent volume. This indicates that the events aren't being ingested and are no longer of value. The task inspects your organization for events that have a `last seen` date in a configurable number of days. For example, events that haven't been ingested in 90 days.
+
+Historical charts or data aren't affected. 
+
+### Clean up single-day events
+This task removes accidental or one-time test events that can negatively impact or clutter up your taxonomy. The task inspects your organization for events that:
+
+* Have the same first seen and last seen dates and;
```

Comment:
I don't believe we have other instances where we use list formatting, but also use the word "and"